src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.css

```diff
@@ -1,20 +1,25 @@
 .header {
-    display: flex;
-    flex-wrap: wrap;
-    font-weight: bold;
+  display: flex;
+  flex-wrap: wrap;
+  font-weight: bold;
 }
 .label {
-    font-weight: bold;
-    margin-bottom: 0px;
+  font-weight: bold;
+  margin-bottom: 0px;
 }
 ul {
-    margin: 5px 0px;
-    padding: 0px;
+  margin: 5px 0px;
+  padding: 0px;
 }
 li {
```

Comment:
testing it right now, but I wonder if the css is scoped to the component only or global.

Reply:
According to my research, css on a module is scoped to the component only, but inherited by parent modules.

Reply:
tried it in a random place and this is what I get:

![image](https://user-images.githubusercontent.com/51387119/106925755-3e9d0380-66c5-11eb-9100-6982918feb9a.png)


![image](https://user-images.githubusercontent.com/51387119/106925802-48bf0200-66c5-11eb-8f60-1814ac2a6a5d.png)

Can you test it ?

Reply:
Confirmed, this is an issue, not just with this document list module, but all modules - it appears that all components that import css affects global css styles.

Added a patch fix the css scope on this one module.